Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -508,7 +508,7 @@ static bool determineIfWeShouldCrashWhenCreatingWebProcess()
 
 #if HAVE(VIDEO_RESTRICTED_DECODING)
 #if PLATFORM(MAC)
-    if (!isFullWebBrowser()) {
+    if (!isFullWebBrowser() || isRunningTest(WebCore::applicationBundleIdentifier())) {
```

Comment:
Why does our test tool evaluate to true for isFullWebBrowser()?

Reply:
That is a good point. I think this was originally done to have WebKitTestRunner exercise the same code paths as Safari for certain tests. An alternative would be to add the isRunningTest check to places where we'd like to use the same code paths.

Thanks for reviewing!